### DISCUSSION TITLE
email id issue

### DISCUSSION THREAD

**Discussion by mithunkumarkundumoon** [community/community#82957] [GitHub Education]
### Hi everyone,

my university does not provide university mail. but when i try to apply for student benefit and choose my university name from the drop down it it asked for university edu mai. and do not let me proceed to go next page. can anyone please resolve this problem??

**Comment by mahfuz0001** [comment]
I suggest you may contact the support team. you can get in youch with them at [GitHub Education Support](https://support.github.com/features/education?tags=dotcom-footer)

**Comment by varunpats** [comment]
Hi @mithunkumarkundumoon,

If college don't provide mail id then there are other different documents which you can use, such as school/college ID.

Please go through [this doc](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) once. Check "What proof do I need to submit for academic verification?" section.

Hope this resolves your problem.
